modules/cats-effect/src/main/scala/PromptsIO.scala

```diff
@@ -22,15 +22,16 @@ import cue4s.*
 class PromptsIO private (
     protected val out: Output,
     protected val terminal: Terminal,
-    protected val colors: Boolean
+    protected val colors: Boolean,
+    protected val windowSize: Int
```

Comment:
Is there a reason to not push windowSize parameter into just the prompts that need it? namely single choice and multi-choice. 

Unlike colors, it's not a universal property shared by all prompts, it's only really applicable to single/multi choice and (future) search prompt

Reply:
wouldn't it make sense to treat it as the universal maximum viewport height? maybe it also makes sense for text input (which does not implement scrolling atm)

Reply:
At this point frankly I don't see a reason for windowSize on input – we don't have the logic for trimming viewport on text input, nor do we have currently any precedent showing it's necessary.

Let's keep it only as part of the prompts for now, and later if necessary we can group them into `Options` and make it more of a first class citizen.

Reply:
ok

Reply:
I think you removed the parameter usages, but kept it in the constructor

Reply:
fixed